packages/frontity/src/actions/unknown.ts

```diff
@@ -1,7 +1,26 @@
+import didYouMean from "didyoumean";
 import { CommanderStatic } from "commander";
 import chalk from "chalk";
 
+/**
+ * @param {String} command - Unknown command
+ * @param {CommanderStatic} program - commander instance
+ * 
+ * @returns {void}
+ */
+
 export default (command: string, program: CommanderStatic) => {
-  console.log(chalk.red(`Unknown command: ${chalk.bold(command)}\n`));
+  console.log(chalk.red(`Unknown command: ${chalk.bold(command)}`));
+  console.log();
+
+  const availableCommands: string[] = program.commands.map(c => c._name);
+  const suggestion: string | string[] = didYouMean(command, availableCommands);
+  
+  if (suggestion) {
+	console.log(`chalk.cyan(Did you mean '${suggestion}'?)`);
```

Comment:
The method call is inside the template string. Just fix this line and it's ready.

```typescript
console.log(chalk.cyan(`Did you mean '${suggestion}'?`));

// instead of console.log(`chalk.cyan(Did you mean '${suggestion}'?)`);
```

Reply:
Oh didn't notice!